io_scene_godot/export_godot.py

```diff
@@ -50,9 +50,7 @@ def find_godot_project_dir(export_path):
     while not os.path.isfile(os.path.join(project_dir, "project.godot")):
         project_dir = os.path.split(project_dir)[0]
         if project_dir in ("/", last):
```

Comment:
Could you explain why these changes?

Reply:
for testing i'm just exporting to /tmp, then i copy to my godot project folder later.
and later i want to have an option to generate the entire godot project folder.

Reply:
Okay, it's fine if you are doing it just for testing purpose, we can revert it before it merge in.

For your information, there is an [exporting option](https://github.com/godotengine/godot-blender-exporter/blob/master/io_scene_godot/__init__.py#L154) that can turn off exporting to a project dir.